README.md

```diff
@@ -28,22 +28,22 @@ Please feel free to contribute by forking and submitting pull requests or by
 submitting feature requests or issues to [issues][githubissues].
 
 ## Requires
-* pytest >= 2.2.3
+* pytest >= 2.9.0
 * jira >= 0.43
 * six
 
 ## Installation
 ``pip install pytest_jira``
 
 ## Usage
-1. Create a `jira.cfg` in the root of your tests
+1. Create a `pytest.ini` in the root of your tests
```

Comment:
this is not fair, to user, you can not change interface. you still have to support jira.cfg and you can print warning that it is deprecated and will be removed in next major version.